### PR TITLE
refactor of graph.h and rewriter.h on types and functions names

### PR DESCRIPTION
**Pull request by JakubPiotrowicz**

### CODE REVIEW

**Comment by JakubPiotrowicz** [comment]
We should discuss moving Graph creation functions from readwriter.h to gaph.h

File: gen-utils.hpp
Comment (dominikstrak2001) [inline gen-utils.hpp:1]: rename to graph_utils.hpp

File: gen-utils.hpp [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#ifndef READWRITER_H_
```
Comment (dominikstrak2001) [inline gen-utils.hpp]: :eyes:

**Review by dominikstrak2001 — APPROVED** [review]
seems sane

**Review by dominikstrak2001 — CHANGES_REQUESTED** [review]
rename gen utils

**Review by dominikstrak2001 — APPROVED** [review]

**Pull request merged**